.env.production

```diff
@@ -1 +1,4 @@
 NEXT_PUBLIC_RUNTIME_ENV=production
+# DSN of pingcap/tug-website
+SENTRY_DSN=https://cc3bce60904940f8a8e5795ef6be2ae7@o226447.ingest.sentry.io/5756513
```

Comment:
Not quite sure whether it should be coded in CircleCI environments: https://circleci.com/docs/2.0/env-vars/?section=pipelines#setting-an-environment-variable-in-a-project